### ISSUE TITLE
Why are there 2 feed discovery sections on the Search page?

### ISSUE THREAD

**Issue by knotbin** [bluesky-social/social-app#7298]
"Recommended" was added to the search page in the recent update and it's completely unnecessary when "Discover New Feeds" is already on that page and has so much better UI.
Having 2 sections to do the same thing makes the app feel cluttered and unpolished. "Recommended" should be removed, it seems it was only added in the first place to reuse the Trending UI.
![image](https://github.com/user-attachments/assets/21fbe479-8e20-4080-9ddc-340b34a04200)
![image](https://github.com/user-attachments/assets/715a096b-4fee-495e-ab92-5004f76ad38a)

**Comment by mschwendener** [comment]
Yes, please let us hide 'Recommended - Feeds we think you might like". 

None of the proposed terms on my search page even have anything to do with my interests.
